### DISCUSSION TITLE
script record search from script file name

### DISCUSSION THREAD

**Discussion by Fluzzroz** [burkybang/SuiteAdvanced-Feedback#77] [Ideas]
I really like how we can search for a script file from the general search. 
I really like how we can find the script record from the script file page.
I would love it even more if we could get to the script record from the script file name in a single step, perhaps by entering the file name in the appropriate sub-search of the global search bar.

**Comment by burkybang** [comment]
Love the idea. I'll give this some thought 🤔

**Comment by burkybang** [comment]
@Fluzzroz This new feature will be released today in version 26.9.0

![image](https://user-images.githubusercontent.com/927886/235800877-a45f663a-68e2-4fb4-9109-47179745908d.png)
